tools/lib/provision.py

```diff
@@ -179,7 +179,11 @@ def main(options):
 
     # setup-apt-repo does an `apt-get update`
     run(["sudo", "./scripts/lib/setup-apt-repo"])
```

Comment:
Should we add L181 as well to the retrial? I guess it run's a script which only installs a single package `lsb-release`.